src/qinfer/resamplers.py

```diff
@@ -61,7 +62,7 @@
 
 class Resampler(with_metaclass(ABCMeta, object)):
     @abstractmethod
-    def __call__(self,  model, particle_weights, particle_locations,
+    def __call__(self,  model, particle_dist,
```

Comment:
As for `particle_mean`, I see what you guys are saying now and agree. The doc for that function should change of course to not reference `f`.